bundles/org.openhab.binding.homeconnect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,664 @@
+	</channel-type>
+	<channel-type id="setpoint_temperature_freezer">
+		<item-type>Number:Temperature</item-type>
+		<label>Freezer temperature</label>
+		<description>Target temperature of the freezer compartment (Range depends on appliance - common range -16 to -24°C).</description>
+		<state step="1" pattern="%.0f %unit%"/>
+	</channel-type>
+	<channel-type id="super_mode_refrigerator">
+		<item-type>Switch</item-type>
+		<label>Refrigerator super mode</label>
+		<description>The setting has no impact on setpoint temperatures but will make the fridge compartment cool to the
+			lowest possible temperature until it is disabled by the manually by the customer or by the HA because of a timeout.</description>
+		<state readOnly="false"/>
+	</channel-type>
+	<channel-type id="super_mode_freezer">
+		<item-type>Switch</item-type>
+		<label>Freezer super mode</label>
+		<description>This setting has no impact on setpoint temperatures but will make the freezer compartment cool to the
+			lowest possible temperature until it is disabled by the manually by the customer or by the home appliance because of
+			a timeout.</description>
+		<state readOnly="false"/>
+	</channel-type>
+	<channel-type id="dryer_drying_target">
+		<item-type>String</item-type>
+		<label>Drying target</label>
+		<description>Specifies the desired dryness setting.</description>
+		<state readOnly="false"/>
+	</channel-type>
+	<channel-type id="hood_venting_level">
+		<item-type>String</item-type>
+		<label>Venting level</label>
+		<description>Current venting level of the hood.</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="hood_intensive_level">
+		<item-type>String</item-type>
+		<label>Intensive level</label>
+		<description>Current venting intensive level of the hood.</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="hood_program_state">
+		<item-type>String</item-type>
+		<label>Program actions</label>
+		<description>Start hood program.</description>
+	</channel-type>
+	<channel-type id="functional_light_state">
+		<item-type>Switch</item-type>
+		<label>Functional light state</label>
+		<description>This setting describes the current functional light state of the home appliance.</description>
+	</channel-type>
+	<channel-type id="functional_light_brightness_state">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Functional light brightness</label>
+		<description>This setting describes the brightness state of the functional light.</description>
+		<state readOnly="false" min="10" max="100" step="1" pattern="%d %unit%"/>
+	</channel-type>
+	<channel-type id="ambient_light_state">
+		<item-type>Switch</item-type>
+		<label>Ambient light state</label>
+		<description>This setting describes the current ambient light state of the home appliance.</description>
+	</channel-type>
+	<channel-type id="ambient_light_brightness_state">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
Just a remark: you are not using Dimmer because the min is 10% ?

Reply:
I've refactored it. I'll ask the community if someone can test the changes because I don't have a device.